content/news/2023-02-22-news-2023-02-22-new-r-ladies-website/index.en.md

```diff
@@ -0,0 +1,107 @@
+---
+title: New R-Ladies Website
+author: Athanasia M. Mowinckel
+date: '2023-02-22'
+slug: news/2023-02-22-new-r-ladies-website
+categories:
+  - R-Ladies
+tags: []
+---
+
+
+We are thrilled to announce the release of our new R-Ladies website!
+
+It's been a long journey, with many people involved, and we are happy to be able to finally share with you this new site and some new content.
+
+<!--more-->
```

Comment:
```suggestion
<!--more-->

The R-Ladies Leadership and Global Team want to express our sincere gratitude to Mo for her incredible work developing the new R-Ladies website. Mo dedicated many hours of hard and invisible work to accomplish this important milestone.

Her technical expertise has been instrumental in bringing this project to fruition. She's put a lot of care into creating a professional and beautiful interface, but also in creating sustainable update workflows for humans and machines. The new website is a significant improvement over the previous one, and we are sure it will enhance our ability to connect with and serve our community of R-Ladies worldwide.
	
Thanks a ton, Mo!

---
```